miscellaneous package install error: unable to find ncurses library headers

When trying to install octave forge package "miscellaneous", the following occurs:

>> pkg install -forge miscellaneous
configure: error: Unable to find ncurses library headers.
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking term.h usability... no
checking term.h presence... no
checking for term.h... no
checking termcap.h usability... no
checking termcap.h presence... no
checking for termcap.h... no
checking ncurses/termcap.h usability... no
checking ncurses/termcap.h presence... no
checking for ncurses/termcap.h... no

pkg: error running the configure script for miscellaneous.
error: called from
    configure_make at line 82 column 9
    install at line 190 column 7
    pkg at line 441 column 9
>>

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -227,6 +227,8 @@ parts:
       - pkg-config
       - python3-venv
       - texinfo
+      - unzip
+      - zip
 
   python-packages:
     plugin: python

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -202,6 +202,7 @@ parts:
       - libglu1-mesa
       - libgraphicsmagick++-q16-12
       - libhdf5-dev
+      - libncurses5-dev
       - libopenblas-dev
       - libpcre3-dev
       - libportaudio2

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -183,7 +183,9 @@ parts:
     stage-packages:
       - build-essential
       - curl
+      - epstool
       - gfortran
+      - ghostscript
       - gnuplot-qt
       - libamd2
       - libarpack2

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -197,6 +197,7 @@ parts:
       - gfortran
       - ghostscript
       - gnuplot-qt
+      - info
       - libamd2
       - libarpack2
       - libcamd2

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -20,6 +20,9 @@ apps:
     desktop: share/applications/org.octave.Octave.desktop
     environment:
       OCTAVE_HOME: "$SNAP"
+      GNUPLOT_DRIVER_DIR: "$SNAP/usr/lib/gnuplot"
+      GNUPLOT_LUA_DIR: "$SNAP/usr/share/gnuplot/gnuplot/5.2/lua"
+      GNUPLOT_PS_DIR: "$SNAP/usr/share/gnuplot/gnuplot/5.2/PostScript"
       LD_LIBRARY_PATH: "$SNAP/lib/octave/$SNAPCRAFT_PROJECT_VERSION"
       LOCPATH: "$SNAP/usr/lib/locale"
       PKG_CONFIG_PATH: "$SNAP/lib/pkgconfig:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig:$SNAP/usr/share/pkgconfig"
@@ -38,6 +41,9 @@ apps:
     command: bin/octave-cli
     environment:
       OCTAVE_HOME: "$SNAP"
+      GNUPLOT_DRIVER_DIR: "$SNAP/usr/lib/gnuplot"
+      GNUPLOT_LUA_DIR: "$SNAP/usr/share/gnuplot/gnuplot/5.2/lua"
+      GNUPLOT_PS_DIR: "$SNAP/usr/share/gnuplot/gnuplot/5.2/PostScript"
       LD_LIBRARY_PATH: "$SNAP/lib/octave/$SNAPCRAFT_PROJECT_VERSION"
       LOCPATH: "$SNAP/usr/lib/locale"
       PKG_CONFIG_PATH: "$SNAP/lib/pkgconfig:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig:$SNAP/usr/share/pkgconfig"
@@ -133,6 +139,8 @@ parts:
       sed -i 's|^prefix=.*|prefix=/usr|' usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig/*.pc
       rm -f usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig/hdf5.pc
       ln -s hdf5-serial.pc usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig/hdf5.pc
+      rm -f usr/bin/gnuplot
+      ln -s gnuplot-qt usr/bin/gnuplot
     prime:
       - -usr/share/texmf/ls-R
     organize:

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -148,6 +148,7 @@ parts:
       - gnuplot-nox
       - gperf
       - icoutils
+      - less
       - libarpack2-dev
       - libcurl4-gnutls-dev
       - libfftw3-dev

--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -27,6 +27,7 @@ apps:
       LOCPATH: "$SNAP/usr/lib/locale"
       PKG_CONFIG_PATH: "$SNAP/lib/pkgconfig:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig:$SNAP/usr/share/pkgconfig"
       PKG_CONFIG_SYSROOT_DIR: "$SNAP"
+      UNITSFILE: "$SNAP/usr/share/units/definitions.units"
     plugs:
       - desktop
       - desktop-legacy
@@ -48,6 +49,7 @@ apps:
       LOCPATH: "$SNAP/usr/lib/locale"
       PKG_CONFIG_PATH: "$SNAP/lib/pkgconfig:$SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/pkgconfig:$SNAP/usr/share/pkgconfig"
       PKG_CONFIG_SYSROOT_DIR: "$SNAP"
+      UNITSFILE: "$SNAP/usr/share/units/definitions.units"
     plugs:
       - desktop
       - desktop-legacy
@@ -238,6 +240,7 @@ parts:
       - pkg-config
       - python3-venv
       - texinfo
+      - units
       - unzip
       - zip
 

